### ISSUE TITLE
Help???

### ISSUE THREAD

**Issue by ghost** [3gstudent/Eventlogedit-Evolution#4]
I installed python and pip but i dont know what to do next. how to enable evtx commands? what to do with the .cpp files on your page? Please help.

**Comment by 3gstudent** [comment]
You need Visual Studio to compile the .cpp files.

**Comment by Lkms19** [comment]
Thank you so much for your response. I will try that when I get home later today.

**Comment by Lkms19** [comment]
Okay, so I installed visual studio and learnt how to work with .cpp files but when I tried to debug one of yours .cpp files i got a host of errors, these are some examples. Any idea how to fix this? Thanks in advanced.

Severity	Code	Description	Project	File	Line	Suppression State
Error (active)	E0167	argument of type "LPWSTR" is incompatible with parameter of type "LPSTR"	

Severity	Code	Description	Project	File	Line	Suppression State
Error	C3861	 'printf': identifier not found	

Severity	Code	Description	Project	File	Line	Suppression State
Error	C3861	 'swprintf': identifier not found

Severity	Code	Description	Project	File	Line	Suppression State
Error	C2664	 'UINT GetSystemDirectoryA(LPSTR,UINT)': cannot convert argument 1 from 'LPWSTR' to 'LPSTR
